### PR TITLE
Document Aidoo devices in Airzone Cloud

### PR DESCRIPTION
**Pull request by Noltari**
## Proposed change
Document Airzone Cloud Aidoo devices.

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/93541
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/airzone_cloud.markdown [hunk 1]
```diff
@@ -15,7 +15,19 @@ ha_codeowners:
...
-This integration interacts with the Cloud API of [Airzone HVAC zoning systems](https://www.airzone.es/en/).
+This integration interacts with the Cloud API of [Airzone devices](https://www.airzone.es/en/).
+
+There are two main types of Airzone devices:
+- [Aidoo](https://www.airzonecontrol.com/aa/en/control-solutions/aidoo/wi-fi/) / [Aidoo Pro](https://www.airzonecontrol.com/aa/en/control-solutions/aidoo/pro/)
+- [Easyzone (US)](https://www.airzonecontrol.com/aa/en/control-solutions/easyzone/) / [Flexa (EU)](https://www.airzonecontrol.com/ib/es/soluciones-de-control/flexa/)
+
+## Aidoo / Aidoo Pro
+
+These devices are wifi controllers that are normally connected to a single air conditioner split system.
```
Comment (frenck) [inline source/_integrations/airzone_cloud.markdown]:
```suggestion
These devices are Wi-Fi controllers that are normally connected to a single air conditioner split system.
```

File: source/_integrations/airzone_cloud.markdown [hunk 2]
```diff
@@ -15,7 +15,19 @@ ha_codeowners:
...
+
+## Easyzone (US) / Flexa (EU)
+
+These devices are connected to ducted air conditioners along with motorized grilles and individual thermostats for every room (zone). Therefore, with a single ducted air conditioning system the user can turn on and off the air conditioner and set different desired temperatures on each room.
```
Comment (frenck) [inline source/_integrations/airzone_cloud.markdown]:
```suggestion
These devices are connected to ducted air conditioners, motorized grilles, and individual thermostats for every room (zone). Therefore, with a single ducted air conditioning system, the user can turn on and off the air conditioner and set different desired temperatures in each room.
```

File: source/_integrations/airzone_cloud.markdown [hunk 3]
```diff
@@ -32,9 +44,15 @@ Password:
 
 ## Sensors
 
+For each Airzone Aidoo (HVAC wifi controller), the following *sensors* are created:
```
Comment (frenck) [inline source/_integrations/airzone_cloud.markdown]:
```suggestion
For each Airzone Aidoo (HVAC Wi-Fi controller), the following *sensors* are created:
```

**Comment by Noltari** [comment]
Thanks @frenck!

**Review by frenck — APPROVED** [review]

**Pull request merged**